orchestration/hca_orchestration/resources/hca_project_config.py

```diff
@@ -10,16 +10,22 @@ class HcaProjectCopyingConfig:
     source_bigquery_region: str
 
 
-@resource({
-    "source_hca_project_id": str,
-    "source_snapshot_name": str,
-    "source_bigquery_project_id": str,
-    "source_bigquery_region": str
-})
+@resource(
+    required_resource_keys={"hca_project_id"},
+    config_schema={
+        "source_snapshot_name": str,
```

Comment:
Note that `hca_project_id` for project copying now comes from a required resource dependency on `hca_project_id`.